bin/runtime/src/lib.rs

```diff
@@ -170,9 +170,25 @@ pub const MAX_BLOCK_SIZE: u32 = 5 * 1024 * 1024;
 parameter_types! {
     pub const Version: RuntimeVersion = VERSION;
     pub const BlockHashCount: BlockNumber = 2400;
-    pub BlockWeights: frame_system::limits::BlockWeights = frame_system::limits::BlockWeights
-        ::with_sensible_defaults(MAX_BLOCK_WEIGHT, NORMAL_DISPATCH_RATIO);
-    pub BlockLength: frame_system::limits::BlockLength = frame_system::limits::BlockLength
+    pub RuntimeBlockWeights: BlockWeights = BlockWeights::builder()
+        .base_block(BlockExecutionWeight::get())
```

Comment:
is this needed?

Reply:
it's verbose, but at the same time lays out all the weights